front/app/components/CityLogoSection/index.tsx

```diff
@@ -49,40 +50,50 @@ const TenantLogo = injectIntl(
 
 interface Props {}
 
-const CityLogoSection = memo(
-  ({ intl: { formatMessage } }: Props & InjectedIntlProps) => {
-    const locale = useLocale();
-    const appConfiguration = useAppConfiguration();
+const CityLogoSection = ({
```

Comment:
removed `memo` here because I was getting linter errors with the HOC injectors and IIRC we've been moving away from using `memo` on every component in general